### PR TITLE
[TSLA-3880] Update tigerastatus samples and custom resource descriptions

### PR DESCRIPTION
**Pull request by jaderhs**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

### CODE REVIEW

File: calico-cloud/get-started/connect/operator-checklist.mdx [hunk 1]
```diff
@@ -47,14 +47,21 @@ $ kubectl get tigerastatus
...
 management-cluster-connection   True        False         False      9m54s
+monitor                         True        False         False      10m
+runtime-security                True        False         False      10m
 ```
 
 If all components show a status of "Available" = TRUE, {{prodname}} is properly installed.
 
+:::note
+The `runtime-security` component will not be displayed if the container threat detection feature is disabled
+:::note
```
Comment (ctauchen) [inline calico-cloud/get-started/connect/operator-checklist.mdx]:
```suggestion
:::note

The `runtime-security` component is available only if [the container threat detection feature is enabled](../../threat/container-threat-detection.mdx#enable-container-threat-detection).

:::note
```

Comment (ctauchen) [reply calico-cloud/get-started/connect/operator-checklist.mdx]: Is runtime-security the only component that isn't enabled by default?

**Review by ctauchen — COMMENTED** [review]
Thanks! A few suggestions:
* see comment about wording and formatting for the note about runtime-security, and apply corresponding changes throughout
* After you submitted your PR, we cut the upcoming 3.16 versioned docs. Please copy your changes to calico-cloud_versioned_docs/version-3.16/get-started/connecct/operator-checklist.mdx (Sorry -- bad timing)

File: calico-cloud/get-started/connect/operator-checklist.mdx (hunk 1 above)
Comment (jaderhs) [reply calico-cloud/get-started/connect/operator-checklist.mdx]: It is at the moment.
Some of the other features, like compliance reports, must be configured in order to use it, which would be similar to enabling it.
runtime-security is the only feature that will not display the tigerastatus depending on its state. All other features will display a tigerastatus regardless of being configured.

**Comment by jaderhs** [comment]
PR rebased on the `main` branch

**Comment by ctauchen** [comment]
lgtm

**Pull request merged**